doc/nrf/protocols/matter/getting_started/tools.rst

```diff
@@ -164,132 +164,119 @@ For more information about the ZAP tool, see the official `ZCL Advanced Platform
 Installing the ZAP tool
 =======================
 
-To install the ZAP tool using the recommended method, complete the following steps:
+.. tabs::
 
-1. Download the ZAP package containing pre-compiled executables and libraries and extract it:
+   .. tab:: Automatic Installation Using the Matter west Command
 
-   a. Open your installation directory for the |NCS| in a command line.
-   #. Navigate to :file:`modules/lib/matter`.
-   #. Run the helper script to download and extract the ZAP package:
+      You can use the Matter West command to automatically download and install the appropriate ZAP tool version dedicated to your current Matter revision.
+      This tool checks the version of the Matter stack, which is located in the default location based on the ``ZEPHYR_BASE`` variable.
+      If you wish to install the ZAP tool for a different Matter version, provide the path to the desired Matter SDK location using the ``-m`` or ``--matter-path`` command argument.
 
-      .. parsed-literal::
-         :class: highlight
+      To install the ZAP tool according to the default Matter SDK location, use the following command:
 
-         python scripts/setup/nrfconnect/get_zap.py -l *zap_location* -o
+      .. code-block:: console
 
-      In this command:
+         west zap-gui
 
-      *  *zap_location* corresponds to the directory where the ZAP tool package is to be extracted.
-      *  The `-o` argument in the command is used to allow overwriting files, if they already exist in the given location.
-         Otherwise the script will display prompt during download and ask for user consent to overwrite the files directly.
+      To install the ZAP tool for a specific Matter version, use the following command and specify the *path_to_Matter_SDK*:
 
-#. Verify the ZAP tool version by comparing the script output with the following possible cases:
+      .. code-block:: console
 
-   * Case 1: If your currently installed ZAP version matches the recommended one, you will see a message similar to the following one:
+         west zap-gui -m <path_to_Matter_SDK>
```

Comment:
This feature is mostly relevant when using Matter in the upstream and less when using Matter in NCS, so it could be added as a `.. note::`.

Reply:
Added a note, but I guess it is the other way around, because it is more relevant in Matter in NCS rather than Matter in Upstream, right?

Reply:
I mean this parameter is only needed when using Matter in upstream.

Reply:
I thought it in a different way, because I thought that somebody may have several copies of sdk-connectedhomeip repos in different places and revisions on their PC, and not especially matter upstream :) but that's right, one of the case is when somebody wants to use Matter upstream repo and west commands. I will change this sentence to explain it better.

Reply:
If one has several NCS installations, one can choose the one using ZEPHYR_BASE, in which case the proper sdk-connectedhomeip will be selected automatically. That's why I thought describing `-m` parameter is only important for the upstream development use case.

Reply:
That's right, that convinces me :D

Reply:
I changed it as @wiba-nordic suggested, I think that keeping it simple is a good way.